swift-coverage/swift-coverage-example/xccov-to-sonarqube-generic.sh

```diff
@@ -16,8 +17,12 @@ function convert_file {
 function xccov_to_generic {
   echo '<coverage version="1">'
   for xccovarchive_file in "$@"; do
-    xcrun xccov view --file-list "$xccovarchive_file" | while read -r file_name; do
-      convert_file "$xccovarchive_file" "$file_name"
+    local xccov_options=""
+    if [[ $xccovarchive_file == *".xcresult"* ]]; then
```

Comment:
Asterisk at the end of the pattern makes this test false positive for Xcode 10 as both
`Build/Logs/Test/*.xcresult/*_Test/*.xccovarchive/` and
`Build/Logs/Test/*.xcresult/` file paths contain `.xcresult`.
Proper test would be `[[ $xccovarchive_file == *".xcresult/" ]]`